src/waveform/renderers/waveformmarkproperties.h

```diff
@@ -0,0 +1,27 @@
+#ifndef WAVEFORMMARKPROPERTIES_H
+#define WAVEFORMMARKPROPERTIES_H
+
+#include <QColor>
+#include <QDomNode>
+
+class SkinContext;
+class WaveformSignalColors;
+
+class WaveformMarkProperties {
+  public:
+    WaveformMarkProperties();
+    WaveformMarkProperties(const QDomNode& node,
+                           const SkinContext& context,
+                           const WaveformSignalColors& signalColors);
+    virtual ~WaveformMarkProperties();
```

Comment:
What if in the future we want to have specialized WaveFormMarks? We will probably subclass WaveFormMarks and also subclass WaveformMarkProperties to add new fields.